repos/spack_repo/builtin/packages/camp/package.py

```diff
@@ -117,7 +120,8 @@ def cmake_args(self):
 
         options.append(self.define_from_variant("ENABLE_HIP", "rocm"))
         if spec.satisfies("+rocm"):
-            options.append("-DHIP_ROOT_DIR={0}".format(spec["hip"].prefix))
+            rocm_root = dirname(spec["llvm-amdgpu"].prefix)
+            options.append("-DROCM_PATH={0}".format(rocm_root))
```

Comment:
```suggestion
            options.append(self.define("ROCM_PATH", rocm_root))
```

Since this is a `CMakePackage` we can use `self.define()` here to generate a `-D` option.